aelnaiem/torronto
# Sauga

We should really start testing now.

To test:
`go build`
`./torronto hostName:portNumber`

We should probably choose what port numbers we want to test on?
